portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -106,6 +108,24 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     List<BlogMessage> BlogMessages = getBlogsFromDatastore();
 
     // Create BlogHashMap Object and put BlogMessages in the map.
+    
+    List<BlogMessage> BlogMessagesReplies = new ArrayList<BlogMessage>();
+    for (BlogMessage message : BlogMessages) {
+      if (message.getParentID() != 0) {
+      BlogMessagesReplies.add(message);
+      BlogMessages.remove(message);
+      }
+    }
```

Comment:
Here too, it's not clear what you want to achieve. Please add a comment to explain

Reply:
lines 110-116

Reply:
I am working on the backend portion for the replies. These lines take the arraylist we have and separate post and replies, then add replies to the messageReplies for its respective post.